test/runtests.jl

```diff
@@ -1,5 +1,15 @@
 using ReTestItems, Test, Pkg
 
+# Used by test/macros.jl, but macros must be defined at global scope (outside `@testset).
+macro foo_test(name)
+    _source = QuoteNode(__source__)
+    quote
+        @testitem $name _source=$_source _run=false begin
+            @test true
+        end
+    end
+end
```

Comment:
For next time: The alternative is that you can put the macro uses inside an `@eval` - that way they won't be macro-lowered until after the macro def has been evaluated.

```julia
julia> @testset "Foo" begin
           @eval macro m() :(2+2) end
           @eval @test @m() == 4
       end
```
but how you have it is nice too

Reply:
that's what i thought, but when i tried that it didn't work
https://github.com/JuliaTesting/ReTestItems.jl/pull/73/commits/71493be6834134db659d75544e924d322a658d67

I guess because i need to `@eval` the `@test` too? why is that? preumably something about not picking up the newly defined name? world-age related? i don't quite understand